src/praisonai-code/praisonai_code/cli/commands/run.py

```
@@ -2143,7 +2211,9 @@ def _run_custom_agent(
             agent_config["auto_save"] = auto_save_name
         
         # Create and run agent
-        _wire_subtree_context_hook(agent_config, no_rules=no_rules)
+        _wire_subtree_context_hook(
+            agent_config, no_rules=no_rules, instructions=instructions
```

Ruff (0.16.0)

[error] Undefined name `instructions`

(F821)